<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>557</width>
    <height>336</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qCodeDesk</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/main/res/qCodeDesk.png</normaloff>:/main/res/qCodeDesk.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>6</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Aktualności</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>3</number>
        </property>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <property name="frameShadow">
             <enum>QFrame::Sunken</enum>
            </property>
            <property name="autoScroll">
             <bool>true</bool>
            </property>
            <property name="indentation">
             <number>0</number>
            </property>
            <property name="uniformRowHeights">
             <bool>false</bool>
            </property>
            <property name="itemsExpandable">
             <bool>true</bool>
            </property>
            <property name="sortingEnabled">
             <bool>false</bool>
            </property>
            <property name="animated">
             <bool>false</bool>
            </property>
            <property name="allColumnsShowFocus">
             <bool>false</bool>
            </property>
            <property name="headerHidden">
             <bool>false</bool>
            </property>
            <attribute name="headerVisible">
             <bool>true</bool>
            </attribute>
            <attribute name="headerCascadingSectionResizes">
             <bool>false</bool>
            </attribute>
            <attribute name="headerDefaultSectionSize">
             <number>50</number>
            </attribute>
            <attribute name="headerHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="headerMinimumSectionSize">
             <number>0</number>
            </attribute>
            <attribute name="headerShowSortIndicator" stdset="0">
             <bool>false</bool>
            </attribute>
            <attribute name="headerStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Temat</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Postów</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ostatnia aktualizacja</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" alignment="Qt::AlignRight">
         <widget class="QToolButton" name="refreshButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Odśwież</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/main/res/icons/refresh.png</normaloff>:/main/res/icons/refresh.png</iconset>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Ustawienia</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>6</number>
          </property>
          <property name="rightMargin">
           <number>6</number>
          </property>
          <item>
           <layout class="QFormLayout" name="formLayout">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="formAlignment">
             <set>Qt::AlignHCenter|Qt::AlignTop</set>
            </property>
            <property name="horizontalSpacing">
             <number>10</number>
            </property>
            <property name="rightMargin">
             <number>6</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="updateIntervalLabel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Częstość aktualizacji</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="updateIntervalSpinBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="wrapping">
               <bool>false</bool>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="suffix">
               <string> min</string>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>360</number>
              </property>
              <property name="value">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item row="1" column="0" alignment="Qt::AlignRight">
         <widget class="QPushButton" name="saveSettingsButton">
          <property name="text">
           <string>Zapisz</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>O programie</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>20</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>173</width>
              <height>173</height>
             </size>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="pixmap">
             <pixmap resource="../resources.qrc">:/main/res/qCodeDesk.png</pixmap>
            </property>
            <property name="scaledContents">
             <bool>true</bool>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>15</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>Ubuntu</family>
                <pointsize>15</pointsize>
               </font>
              </property>
              <property name="text">
               <string>qCodeDesk rev1h</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>CodeDesk (cpp0x.pl) w wersji dla systemów Linux i Windows.

Licencja: GPLv3</string>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignHCenter|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>© 2014 MrPoxipol &amp;lt;&lt;a href=&quot;http://poxip.ml&quot;&gt;poxip.ml&lt;/a&gt;&amp;gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignBottom|Qt::AlignHCenter</set>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/main/res/icons/exit.png</normaloff>:/main/res/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Zakończ</string>
   </property>
   <property name="toolTip">
    <string>Kończy działanie aplikacji</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/main/res/icons/refresh.png</normaloff>:/main/res/icons/refresh.png</iconset>
   </property>
   <property name="text">
    <string>Odśwież</string>
   </property>
   <property name="toolTip">
    <string>Aktualizuje listę tematów</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R, F5</string>
   </property>
  </action>
  <action name="actionToggleWindow">
   <property name="icon">
    <iconset resource="../resources.qrc">
     <normaloff>:/main/res/icons/window.png</normaloff>:/main/res/icons/window.png</iconset>
   </property>
   <property name="text">
    <string>Pokaż/Schowaj okno</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
